shared_models/shared_models.py

```diff
@@ -137,9 +137,10 @@ class CredentialExchange(BaseModel):
         "request-received",
         "credential-issued",
         "credential-received",
+        "credential-acked",
```

Comment:
I don't think we need this (and otherwise we need to fix this) as credential-acked is transformed to done. this is to abstract away the differences between v1 and v2

Reply:
See: https://github.com/didx-xyz/aries-cloudapi-python/blob/46cdf73703b271c29bd8b3a36871d78b211177f1/shared_models/shared_models.py#L302

in v2 `credential-acked` does not exist